hexa/pipelines/models.py

```diff
@@ -340,6 +340,37 @@ def is_schedulable(self):
         elif self.type == PipelineType.ZIPFILE:
             return self.last_version and self.last_version.is_schedulable
 
+    def get_config_from_previous_version(self, new_parameters: dict):
+        """
+        Get the config from the previous version of the pipeline considering only common parameters between the new and the previous version.
+        """
+        previous_config_from_common_parameter = {}
+        if self.last_version:
+            previous_parameters = self.last_version.parameters
+            common_parameters = [
```

Comment:
I have only one comment on the naming, that "common" might not reflect the best that we are talking about overlapping parameters between new and previous versions

Reply:
Makes a lot of sense, thanks, fixed in next commits